core/domain/topic_jobs_one_off_test.py

```diff
@@ -125,7 +125,7 @@ def test_migration_job_converts_old_topic(self):
             self.TOPIC_ID, self.albert_id, 'A name', 'abbrev',
             'a name', '', [], [], [], 2)
         topic = (
-            topic_fetchers.get_topic_by_id(self.TOPIC_ID))
+            topic_models.TopicModel.get(self.TOPIC_ID))
```

Comment:
Same thing, not sure if this should be changed. Please double-check other cases of this as well.

Reply:
See [comment](https://github.com/oppia/oppia/pull/8498#discussion_r380351818).